src/renderer/app-update-warning/app-update-warning.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { makeObservable, observable } from "mobx";
+import { UpdateAvailableChannel } from "../../common/ipc";
+import { Singleton } from "../utils";
+import type { IpcRenderer } from "electron";
+
+interface Dependencies {
+  readonly ipcRenderer: IpcRenderer;
+  readonly sessionStorage: Storage;
```

Comment:
Hmm. Shouldn't this be `localStorage` instead of `sessionStorage`? I'm not sure.